tests/samples/issue-546-editable-subproject/src/pkg/__init__.py

```diff
@@ -0,0 +1 @@
+from ._pkg import *   # noqa: F403,F401
```

Comment:
```suggestion
from ._core import logic
```
?

Reply:
I'm not sure I'm understand the idea of this change. 
`_core` has been renamed to `subpkg` in the meantime to make clear that this is just a standalone sub package. So if any we should add `from ._subpkg import *`. But there is nothing to export in `subpkg` so this wouldn't change much I guess. 
The extension `_pkg` is installed in the module folder `pkg` and exposes currently only the name `LogicCPP`. I can change the star import to that if you want, but I think the linter would still complain without the ignore codes. 
In general, feel free to change the test case however you see fit. This case is simply the one I managed to make skbuild fail the editable install with.

Reply:
I have instead renamed the pybind11 module to _pkg as well (since I renamed the test module to pkg and subpkg with the 'internal' lib being _pkg). Would changing the star import be important?